tools/docker/build_docker_inference.py

```
@@ -0,0 +1,59 @@
+"""
```

Ruff (0.14.4)

The file is executable but no shebang is present

(EXE002)